src/autoqasm/api.py

```diff
@@ -323,6 +324,12 @@ def _convert_subroutine(
     with aq_program.build_program() as program_conversion_context:
         oqpy_program = program_conversion_context.get_oqpy_program()
 
+        # Iterate over list of dictionary keys to avoid runtime error
+        for key in list(kwargs):
+            is_keyword, new_name = is_reserved_keyword(key)
+            if is_keyword:
+                kwargs[new_name] = kwargs.pop(key)
```

Comment:
[nit] the `if is_keyword:` check is not really necessary - you could just always execute this line.

Reply:
Right!